testing_and_setup/compass/ocean/global_ocean/config_files/config_initial_state.xml

```diff
@@ -52,6 +54,14 @@
 		<step executable="gpmetis">
 			<argument flag="graph.info">8</argument>
 		</step>
+		<step executable="./make_vertical_grid.py">
```

Comment:
@xylar, I've only tested the QU240 right now, but this file changes all of them. Is it best to call the python script as an command-line executable here? That's what I'm most familiar with.

Reply:
Yes, this is fine.